### PR TITLE
Strings file errors removed 

### PR DESCRIPTION
**Pull request by anmol-Git**
some strings are not available in the bengali translation now have been added and some error have been resolved

### CODE REVIEW

**Review by calren — APPROVED** [review]
Awesome, thanks for this @anmol-Git !

File: app/src/main/res/values-bn/strings.xml [hunk 1]
```diff
@@ -29,16 +29,18 @@
...
     <string name="menu_item_share_plant">শেয়ার</string>
     <string name="share_text_plant">অ্যানড্রয়েড সানফ্লাওয়ার অ্যাপে আমার %s গাছ টি দেখুন</string>
+    <string name="gallery_title" >আনস্প্ল্যাশ দ্বারা ফটো</string>
+    <string name="gallery_content_description" >গ্যালারী স্ক্রিনে নেভিগেট করুন</string>
 
     <!-- String plurals and their related prefix / suffix strings -->
     <string name="watering_needs_prefix">জল দেয়া প্রয়োজন</string>
     <plurals name="watering_needs_suffix">
-        <item quantity="one">প্রতি দিন</item>
+        <item quantity="one">প্রতি %d দিন</item>
```
Comment (calren) [inline app/src/main/res/values-bn/strings.xml:38]: I'm not an expert in this language, but the one quantity, I assume we don't need a placeholder for the quantity?

File: app/src/main/res/values-bn/strings.xml [hunk 2]
```diff
@@ -29,16 +29,18 @@
...
         <item quantity="other">প্রতি %d দিন</item>
     </plurals>
 
     <plurals name="watering_next">
-        <item quantity="one">আগামী কাল জল দিন।</item>
+        <item quantity="one">আগামী কাল %d জল দিন।</item>
```
Comment (calren) [inline app/src/main/res/values-bn/strings.xml:43]: same comment about not needing a placeholder for the one quantity?

**Review by calren — CHANGES_REQUESTED** [review]
some questions about the 'one' quantity strings

File: app/src/main/res/values-bn/strings.xml (hunk 2 above)
Comment (anmol-Git) [reply app/src/main/res/values-bn/strings.xml:43]: Yes indeed ,sorry for the mistake i will correct it

**Comment by riggaroo** [comment]
Hi, 
Thanks for your contribution, however Sunflower is going to be archived in favour of maintaining a smaller set of up-to-date set of compose-samples, that can be found here: https://github.com/android/compose-samples, as well as snippets that can be found here: https://github.com/android/snippets.

Please feel free to keep your fork up-to-date for your learning, but this repository will be archived shortly.

**Pull request closed without merging**